[Workers] tutorials style guide revision (#3199)
cloudflare/cloudflare-docs: products/workers/src/content/tutorials/deploy-a-static-wordpress-site/index.md

## Prerequisites

This tutorial assumes that you are:

- the Administrator account on your WordPress site.
- able to install WordPress Plugins on the site.

## Setup

To start,  install the [WP2Static](https://wordpress.org/plugins/static-html-output-plugin/) plugin to export your WordPress site to a zip file. In your WordPress admin, navigate to the **Add Plugins** page, by default at `/wp-admin/plugin-install.php`. In the search bar, search WP2Static and confirm that the resulting plugin that you will be installing matches the plugin below.

![Installing the plugin](./media/wordpress--install-plugin.png)

Select **Install** on the plugin. After it has finished installing, select **Activate**.

### Export your WordPress site

> When your application refreshes, you should see a new section of your sidebar called WP2Static. Navigate to that section to begin your first static WordPress export.

Reviewer: ```suggestion
When your application refreshes, you should see a new section in your sidebar called **WP2Static**. Navigate to that section to begin your first static WordPress export.
```

> From the **Where will you host..** dropdown on the page, select _ZIP archive (.zip)_. WP2Static supports a number of different export types, including manual and automated exports. To make deployment easier, select **Allow offline usage** – this will make all the URLs in your WordPress export relative: `/logo.png` versus `https://mysite.com/logo.png`, and will reduce the number of asset errors that could occur depending on your theme and WordPress setup.

Reviewer: ```suggestion
From the **Where will you host...** dropdown on the page, select _ZIP archive (.zip)_. WP2Static supports a number of different export types, including manual and automated exports. To make deployment easier, select **Allow offline usage** – this will make all the URLs in your WordPress export relative: `/logo.png` versus `https://mysite.com/logo.png`, and will reduce the number of asset errors that could occur depending on your theme and WordPress setup.
```

> To do your first export, select **Start static site export** (this may take a while), and when it completes, save the ZIP file somewhere you can easily find it later.

Reviewer: ```suggestion
To do your first export, select **Start static site export** (this operation may take a while), and when it completes, save the ZIP file somewhere you can easily find it later.
```

![Exporting your WordPress site](./media/wordpress--export.png)

## Creating the Workers project

> With an export ready of our site, it is time to get to work deploying it. To do this, use [Wrangler](https://github.com/cloudflare/wrangler), the command-line tool for Cloudflare Workers. If you have not yet installed and configured Wrangler, refer to the [Get started](/get-started/guide) guide.

Author: ```suggestion
With an export ready of your site, it is time to get to work deploying it. To do this, use [Wrangler](https://github.com/cloudflare/wrangler), the command-line tool for Cloudflare Workers. If you have not yet installed and configured Wrangler, refer to the [Get started](/get-started/guide) guide.
```

> Once Wrangler is installed and configured, we can create a new project for deploying our static WordPress site to Workers. To generate a new project run:

Author: ```suggestion
Once Wrangler is installed and configured, we can create a new project for deploying your static WordPress site to Workers. To generate a new project run:
```

$ wrangler generate --site wp-static
```

The [`--site`](/platform/sites/configuration) flag indicates that you want to deploy a static site: your static WordPress site.

### Port the WordPress Site

The newly generated `wp-static` directory will contain three components:

1. A `public` directory, representing the site you want to deploy to Workers. This will be the location of your WordPress site.

> 2. A `workers-site` directory, which contains the Workers script that will serve your website content. In this tutorial, you will not change anything in this folder.
> 3. A `wrangler.toml` file, which contains the configuration details for our `wp-static` file. You will populate this file with some information before the project is deployed.

Reviewer: ```suggestion
1. A `workers-site` directory, which contains the Workers script that will serve your website content. In this tutorial, you will not change anything in this folder.
1. A `wrangler.toml` file, which contains the configuration details for your `wp-static` directory. You will populate this file with some information before the project is deployed.
```

## Deploying

> To preview and deploy our application, fill out the fields in your `wrangler.toml` file — the configuration file for this project. Fill out the [`account_id`](/get-started/guide#6a-obtaining-your-account-id-and-zone-id) field in `wrangler.toml` with your Cloudflare account ID.

Reviewer: ```suggestion
To preview and deploy your application, fill out the fields in your `wrangler.toml` file — the configuration file for this project. Fill out the [`account_id`](/get-started/guide#6a-obtaining-your-account-id-and-zone-id) field in `wrangler.toml` with your Cloudflare account ID.
```

Using Wrangler’s preview feature, you can quickly upload a version of your site to the Cloudflare Workers preview service and review that the static export looks as expected. Running `wrangler dev` will upload your static site and preview it in a browser window.

When your site looks correct in Wrangler’s preview, you can move onto publishing your project to a domain. For a guide on how to do this, refer to [Get started](/get-started/guide#6-configure-your-project-for-deployment).

> [![Demo site](./media/wordpress--demo.png)](https://wp-static.signalnerve.workers.dev)

Reviewer: I never used an image as the only content of a link. It may not be obvious that the link will not expand the image to its full size, as usual, but will instead navigate to an external site. I'd check for possible issues in a local build.
